docs/source/_toctree.yml

```diff
@@ -29,6 +29,8 @@
     title: Distributing Training
   - local: use_model
     title: Using Trained Models
+  - local: jobs_training
```

Comment:
Can you place it between Command Line Interface (CLI) and Customizing the Training instead?